efficient_full_table_scan.go

```diff
@@ -0,0 +1,200 @@
+// THIS SCRIPT ASSUMES USING A SINGLE CLIENT TO RUN THE FULL TABLE SCAN
+// Prerequisite:
+//  Install go 1.7: https://www.digitalocean.com/community/tutorials/how-to-install-go-1-7-on-centos-7
+//  Install gocql driver: https://academy.datastax.com/resources/getting-started-apache-cassandra-and-go
+
+package main
+
+import (
+        "fmt"
+        "math"
+        "math/rand"
+        "os"
+        "sync"
+        "sync/atomic"
+        "time"
+
+        "github.com/gocql/gocql"
+)
+
+// Your Scylla cluster setup
+const (
+        // NodesInCluster - the number of nodes in your cluster
+        NodesInCluster = 3
+        // CoresInNode - the number of cores in each node
+        CoresInNode = 8
+        // SmudgeFactor - is a factor to add to make parallelism cooler
+        SmudgeFactor = 3
+        // NumberOfParallelClientThreads - the calculated number of parallel threads the client should run
+        NumberOfParallelClientThreads = NodesInCluster * CoresInNode * SmudgeFactor
```

Comment:
consider extracting the values to an external JSON file
example:
https://groups.google.com/forum/?fromgroups=#!topic/golang-nuts/sAofmg2_lL8